tests/test_invoice.py

```diff
@@ -170,57 +172,61 @@ def test_equality(self):
 
 class TestInvoiceReq:
     async def test_send_required_args_only(self, bot, chat_id, provider_token):
-        message = await bot.send_invoice(
-            chat_id=chat_id,
-            title=Space.title,
-            description=Space.description,
-            payload=Space.payload,
-            provider_token=provider_token,
-            currency=Space.currency,
-            prices=Space.prices,
+        send_inv_task = asyncio.create_task(
+            bot.send_invoice(
+                chat_id=chat_id,
+                title=Space.title,
+                description=Space.description,
+                payload=Space.payload,
+                provider_token=provider_token,
+                currency=Space.currency,
+                prices=Space.prices,
+            )
+        )
+        create_inv_link_task = asyncio.create_task(
+            bot.create_invoice_link(
+                title=Space.title,
+                description=Space.description,
+                payload=Space.payload,
+                provider_token=provider_token,
+                currency=Space.currency,
+                prices=Space.prices,
+            )
```

Comment:
please revert the changes for this tests, the tasks are not gathered and this decreases readeability IMO